examples/step-14/step-14.cc

```diff
@@ -709,22 +710,22 @@ namespace Step14
                       ConstraintMatrix &)
         = &DoFTools::make_hanging_node_constraints;
 
-      Threads::Task<> side_task
-        = Threads::new_task (mhnc_p,
-                             dof_handler,
-                             hanging_node_constraints);
+      // Start a side task then continue on the main thread
+      Threads::Task<> side_task(std_cxx11::bind(mhnc_p,std_cxx11::cref(dof_handler),
+                                                std_cxx11::ref(hanging_node_constraints)));
```

Comment:
This is certainly the C++11 way of writing the code, but any reason to chose the syntax here?